### ISSUE TITLE
the two options "Report line indentation" and "Report paragraph indentation" cannot be save as a individual profile.

### ISSUE THREAD

**Issue by vgjh2005** [nvaccess/nvda#5910]
Hi
I am using NVDA next 13300. Step to reproduce:
1. Create a profile for a application( notepad etc). Please be sure the profile is activated.
2. Change the state of line indentation or Report paragraph indentation".
3. Switch to the configuration profile normal configuration
4. Open the document format dialog, and check the state of the two options.
 The two options state are also changed.
Thanks!

**Comment by derekriemer** [comment]
I think there are actually a number of config settings in which this bug 
occurs.


On 4/28/2016 6:35 AM, vgjh2005 wrote:
>
> Hi
> I am using NVDA next 13300. Step to reproduce:
> 1. Create a profile for a application( notepad etc). Please be sure 
> the profile is activated.
> 2. Change the state of line indentation or Report paragraph indentation".
> 3. Switch to the configuration profile normal configuration
> 4. Open the document format dialog, and check the state of the two 
> options.
> The two options state are also changed.
> Thanks!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/nvaccess/nvda/issues/5910>
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194

**Comment by jcsteh** [comment]
When the application profile is active, it is triggered. You can't switch to another profile; you can manually activate one and any settings therein will override application profiles. However, the normal configuration can never override other profiles; it is the configuration on which everything else is based. You'll notice that when an application profile is active, there is no manual activate button for the normal configuration.

So, in short, the reason the settings aren't changing when you say you're switching to the normal configuration is that you aren't (and can't) switch to the normal configuration at all. If you don't want your application profile to trigger, either delete it or temporarily disable triggers.
